packages/core/src/hydration/event_replay.ts

```diff
@@ -79,3 +118,78 @@ export function setJSActionAttribute(
     }
   }
 }
+
+function setEventReplayer(dispatcher: Dispatcher) {
+  dispatcher.setEventReplayer(queue => {
+    for (const event of queue) {
+      handleEvent(event);
+    }
+  });
+}
```

Comment:
I've added some docs, see below. Could you please confirm whether event replay is kicked off as a side-effect of a callback registration?
```suggestion
/**
 * Registers a function that should be invoked to replay events.
 * Event replay is kicked off as a side-effect of registering this function.
 */
function setEventReplayer(dispatcher: Dispatcher) {
  dispatcher.setEventReplayer(queue => {
    for (const event of queue) {
      invokeEventHandlerFor(event);
    }
  });
}
```

Reply:
i believe it's done via the "registerDispatcher" function next to it. I've left a comment.